config/apidoc.php

```diff
@@ -9,6 +9,11 @@
      */
     'type' => 'static',
 
+    /*
+     * Static output folder: Uncomment and change it if you need the documentation to be generated in a different folder
```

Comment:
* Static output folder: HTML documentation and assets will be generated in this folder